internal/services/streamanalytics/stream_analytics_output_servicebus_queue_resource_test.go

```diff
@@ -415,7 +431,30 @@ resource "azurerm_stream_analytics_output_servicebus_queue" "import" {
 `, template)
 }
 
-func (r StreamAnalyticsOutputServiceBusQueueResource) template(data acceptance.TestData) string {
+func (r StreamAnalyticsOutputServiceBusQueueResource) authenticationMode(data acceptance.TestData, identity string) string {
+	template := r.template(data, identity)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_stream_analytics_output_servicebus_queue" "test" {
+  name                      = "acctestinput-%d"
+  stream_analytics_job_name = azurerm_stream_analytics_job.test.name
+  resource_group_name       = azurerm_stream_analytics_job.test.resource_group_name
+  queue_name                = azurerm_servicebus_queue.test.name
+  servicebus_namespace      = azurerm_servicebus_namespace.test.name
+  shared_access_policy_key  = azurerm_servicebus_namespace.test.default_primary_key
+  shared_access_policy_name = "RootManageSharedAccessKey"
+
+  serialization {
+    type     = "Json"
+    encoding = "UTF8"
+    format   = "LineSeparated"
+  }
+}
+`, template, data.RandomInteger)
+}
+
+func (r StreamAnalyticsOutputServiceBusQueueResource) template(data acceptance.TestData, identity string) string {
```

Comment:
identity is unused in this function?

Reply:
it's used on L435.

Reply:
life 435 doesn't exist in this function? 
<img width="1262" alt="image" src="https://user-images.githubusercontent.com/1638467/192077434-9b7b0f1a-ab1e-4a6f-ae80-eef6006a8213.png">

Reply:
Sorry, misunderstood previously. Added `identity` usage in template now.